hasura/metadata/databases/default/tables/public_vaults.yaml

```diff
@@ -48,4 +48,7 @@ select_permissions:
             - profile:
                 id:
                   _eq: X-Hasura-User-Id
+            - circle:
```

Comment:
you should add this checks further up in the relationships filter, where `circles` is already.

instead of:
```
circles:
  users:
```

you'll end up with something like:
```
circles:
  _and:
    deleted_at:
      _is_null: true
    users:
```